rainier-core/src/main/scala/com/stripe/rainier/compute/Recording.scala

```diff
@@ -0,0 +1,35 @@
+package com.stripe.rainier.compute
+
+import java.io._
+
+trait Recording {
```

Comment:
why make this a trait instead of just having the case class be called `Recording` for now? As in, what alternative implementations of the trait do you imagine there will be?